shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/main/java/org/apache/shardingsphere/data/pipeline/api/job/progress/JobProgress.java

```diff
@@ -101,9 +88,6 @@ public int getInventoryFinishedPercentage() {
      * @return latest active time, <code>0</code> is there is no activity
      */
     public long getIncrementalLatestActiveTimeMillis() {
-        List<Long> delays = incrementalTaskProgressMap.values().stream()
-                .map(each -> each.getIncrementalTaskDelay().getLatestActiveTimeMillis())
-                .collect(Collectors.toList());
-        return delays.stream().reduce(Long::max).orElse(0L);
+        return incremental == null ? 0L : incremental.getIncrementalLatestActiveTimeMillis();
```

Comment:
`var == null` could be replaced to `null == var`, there're 4 places could be improved.

Reply:
get it